helix-core/src/test/java/org/apache/helix/ThreadLeakageChecker.java

```diff
@@ -0,0 +1,181 @@
+package org.apache.helix;
+
```

Comment:
@kaisun2000 Can you add Apache License ?

Reply:
added.